modules/layers/src/contour-layer/contour-layer.js

```diff
@@ -35,11 +36,13 @@ const defaultProps = {
   // grid aggregation
   cellSize: {type: 'number', min: 1, max: 1000, value: 1000},
   getPosition: {type: 'accessor', value: x => x.position},
+  getWeight: {type: 'accessor', value: x => 1},
 
   // contour lines
   contours: [{threshold: DEFAULT_THRESHOLD}],
 
-  fp64: false
+  fp64: false,
```

Comment:
Just curious, does the aggregator support fp64?

Reply:
Yeah, it supports.